.gitignore

```diff
@@ -158,6 +158,14 @@ third_party/nvml/
 third_party/logcabin/
 
 #==============================================================================#
+<<<<<<< HEAD
```

Comment:
This seems like a merge conflict. It's better to resolve it before pull request.